cypress/integration/brewery/ScenarioParametersValidation.spec.js

```diff
@@ -89,4 +89,22 @@ describe('scenario parameters inputs validation', () => {
       });
     }
   );
+  it('checks error messages for min and max values in number input', () => {
+    ScenarioParameters.expandParametersAccordion();
+    BreweryParameters.getStockInput().clear().type('150');
+    BreweryParameters.getStockHelperText().should('be.visible').contains('Maximum value');
+    BreweryParameters.getStockInput().clear().type('-150');
+    BreweryParameters.getStockHelperText().should('be.visible').contains('Minimum value');
+    BreweryParameters.getRestockInput().clear().type('30');
+    BreweryParameters.getRestockHelperText().should('be.visible').contains('Maximum value');
+    BreweryParameters.getRestockInput().clear().type('-30');
+    BreweryParameters.getRestockHelperText().should('not.exist');
+    BreweryParameters.getWaitersInput().clear().type('-5');
+    BreweryParameters.getWaitersHelperText().should('be.visible').contains('Minimum value');
+    BreweryParameters.getWaitersInput().clear().type('25');
+    BreweryParameters.getWaitersHelperText().should('not.exist');
+    ScenarioParameters.getSaveButton().should('be.disabled');
+    ScenarioParameters.getLaunchButton().should('be.disabled');
+    ScenarioParameters.discard();
```

Comment:
Maybe add tests to check that save and launch button become enabled if all values are validated ?

Reply:
Yes, good idea, i'll implement this check ! :+1: